terraform-provider-constellation/internal/provider/provider.go

```diff
@@ -28,7 +28,7 @@ type ConstellationProviderModel struct{}
 // ConstellationProvider is the provider implementation.
 type ConstellationProvider struct {
 	// version is set to the provider version on release, "dev" when the
-	// provider is built and ran locally, and "test" when running acceptance
+	// provider is built and ran locally, and "v2.13.0" when running acceptance
```

Comment:
```suggestion
	// provider is built and ran locally, and the latest Constellation release version when running acceptance
```
When injecting the version dynamically.